test/fw/ptl/utils/pbs_logutils.py

```diff
@@ -286,20 +287,51 @@ def open_log(self, log, hostname=None, sudo=False):
         :type sudo: boolean
         :returns: A file instance
         """
+        readcmd = ['cat', log]
+        taillogs = 10000
+        if start:
+            i = 0
+            while(True):
```

Comment:
This function works on log files on machines other than this one (the hostname parameter).  This code expects the log file to be local.